### DISCUSSION TITLE
Pre-complied "Display" bin - was USE_DISPLAY_MODES1TO5 set?

### DISCUSSION THREAD

**Discussion by Sum-Guy** [arendst/Tasmota#17564] [Q&A]
Just curious if the setting

USE_DISPLAY_MODES1TO5

would have been set for any or all of the precompiled Display.bin tasmota binaries.

**Comment by sfromis** [comment]
You can check features included in your current binary by running the tool [decode-status](https://github.com/tasmota/decode-status/). And the answer should be "yes".....

You can also see the setup here:
https://github.com/arendst/Tasmota/blob/development/tasmota/include/tasmota_configurations.h#L274-L358
